app/default/src/Controller/DefaultController.php

```diff
@@ -28,8 +34,11 @@ public function list(
 		AdditionalServiceRepositoryInterface $serviceRepository,
 		Pagination                           $pagination,
 		Criteria                             $criteria
+		
 	): Response
 	{
+		// $this->logger = $logger;
```

Comment:
@212223 looks it can be removed (and the constructor and property as well) 😉

Reply:
Thank you for spotting it. Done in [59e47a7](https://github.com/lrynek/painless-car-rental/pull/1/commits/59e47a75572e500cd7070e0f4740bb8ed76843cd)